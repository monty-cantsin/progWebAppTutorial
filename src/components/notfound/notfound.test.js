import React from 'react'
import ReactDOM from 'react-dom'
import ReactTestRender from 'react-test-renderer'
import toJson from 'enzyme-to-json'
import { shallow } from 'enzyme'

import NotFound from './NotFound'

test('NotFound component should render as expected', () => {
	const component = shallow(<NotFound />)
	const tree = toJson(component)

	expect(tree).toMatchSnapshot()
})