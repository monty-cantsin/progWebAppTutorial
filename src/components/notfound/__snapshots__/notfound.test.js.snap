// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`NotFound component should render as expected 1`] = `
<h1>
  Uh oh! Looks like you've taken a wrong turn!
</h1>
`;
